deployments/Dockerfiles/tendermint/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
+# Use this script to test if a given TCP host/port are available
```

Comment:
We can remove this script after https://github.com/valory-xyz/open-autonomy/issues/1712